ranger/default.go

```diff
@@ -359,7 +352,7 @@ func DefaultRouter() RangerOption {
 			r := router.NewRouter(rng.env.String())
 			r.OnEveryRequest(mws...)
 			r.HandleNotFound(http.HandlerFunc(func(wx http.ResponseWriter, rx *http.Request) {
-				if strings.Index(rx.Header.Get("Accept"), "text/html") >= 0 && rx.URL.Path != rng.url.Path {
+				if strings.Contains(rx.Header.Get("Accept"), "text/html") && rx.URL.Path != rng.url.Path {
```

Comment:
This is different behavior and more permissive than `Index`. What's the reason for the change?

Reply:
I can bump it back, just linter change

Reply:
I'm not tracking what you mean. Can you provide an example where the code would do something different. Contains is:

```
Index(s, substr) >= 0
```

under the hood

Reply:
Ah, yes, my mistake. I meant for the comparison to be `strings.Index(...) == 0`. Oh well, we shouldn't change the behavior now, so `Contains` is correct.